charmcraft/helptexts.py

```diff
@@ -63,16 +63,18 @@ def _build_item(title, text, title_space):
     The title starts in column 4 with an extra ':'. The text starts in
     4 plus the title space; if too wide it's wrapped.
     """
-    text_space = TERMINAL_WIDTH - title_space - 4
+    # wrap the general text to the desired max width (discounting the space for the title,
+    # the first 4 spaces, the two spaces to separate title/text, and the ':'
+    text_space = TERMINAL_WIDTH - title_space - 7
```

Comment:
Not a huge fan of magic constants (7 here) but Oh well :-)

Reply:
It's a good idea to isolate that 7 in a variable, though, as it's used more than once.